src/Stoa.ts

```diff
@@ -114,6 +115,7 @@ class Stoa extends WebService
         this.app.get("/transaction/pending/:hash", this.getTransactionPending.bind(this));
         this.app.get("/utxo/:address", this.getUTXO.bind(this));
         this.app.get("/transaction/status/:hash", this.getTransactionStatus.bind(this));
+        this.app.get("/transaction/fees/:size", this.getTransactionFees.bind(this));
```

Comment:
```suggestion
        this.app.get("/transaction/fees/:tx_size", this.getTransactionFees.bind(this));
```
It's easier to understand.